tests/integration/session/test_questionnaire_store_upgrade.py

```diff
@@ -28,15 +28,15 @@ class TestLogin(IntegrationTestCase):
 
     def test_questionnaire_store_is_upgraded(self):
         # Given
-        previous_version = QuestionnaireStore.LATEST_VERSION - 1
 
-        # Creates a QuestionnaireStore with previous version
-        with patch('app.data_model.questionnaire_store.QuestionnaireStore.get_latest_version_number', return_value=previous_version):
-            self.launchSurvey('test', '0205')
+        self.launchSurvey('test', '0205')
```

Comment:
Does this need to use test_0205.json? Ideally we should be getting rid of these and using actual test schemas. Not a blocker as it's not related to this PR.

Reply:
Fixed now.